pallet/src/weights.rs

```diff
@@ -33,32 +33,27 @@
 use frame_support::{traits::Get, weights::Weight};
 use core::marker::PhantomData;
 
+// Note: The weights have not only been generated by the Substrate benchmarks with a template-node.
+//       A couple of manual and additional modifications have been done which are marked in the
+//       comments.
+
 /// Weight functions for `pallet_move`.
 pub struct SubstrateWeight<T>(PhantomData<T>);
 impl<T: frame_system::Config> crate::weight_info::WeightInfo for SubstrateWeight<T> {
-	/// Storage: `MoveModule::MultisigStorage` (r:1 w:1)
-	/// Proof: `MoveModule::MultisigStorage` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// Storage: `Balances::Locks` (r:3 w:3)
-	/// Proof: `Balances::Locks` (`max_values`: None, `max_size`: Some(1299), added: 3774, mode: `MaxEncodedLen`)
-	/// Storage: `Balances::Freezes` (r:3 w:0)
-	/// Proof: `Balances::Freezes` (`max_values`: None, `max_size`: Some(49), added: 2524, mode: `MaxEncodedLen`)
-	/// Storage: `System::Account` (r:4 w:4)
-	/// Proof: `System::Account` (`max_values`: None, `max_size`: Some(128), added: 2603, mode: `MaxEncodedLen`)
-	/// Storage: `MoveModule::VMStorage` (r:2 w:1)
-	/// Proof: `MoveModule::VMStorage` (`max_values`: None, `max_size`: None, mode: `Measured`)
-	/// The range of component `n` is `[0, 7]`.
+	/// The range of component `n` is `[0, 24]`.
 	fn execute(n: u32, ) -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `9479`
-		//  Estimated: `17285 + n * (1902 ±16)`
-		// Minimum execution time: 33_000_000 picoseconds.
-		Weight::from_parts(33_000_000, 0)
-			.saturating_add(Weight::from_parts(0, 17285))
-			// Standard Error: 70_255_180
-			.saturating_add(Weight::from_parts(989_536_443, 0).saturating_mul(n.into()))
-			.saturating_add(T::DbWeight::get().reads(5))
-			.saturating_add(T::DbWeight::get().writes(3))
-			.saturating_add(Weight::from_parts(0, 1902).saturating_mul(n.into()))
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 46_000_000 picoseconds.
+		Weight::from_parts(44_730_881, 0)
+			.saturating_add(Weight::from_parts(0, 0))
+			// Standard Error: 9_543
+            // Manually added divisor of 20:
+			.saturating_add(Weight::from_parts(13_002_180, 0).saturating_mul((n/20).into()))
```

Comment:
maybe this is a better approach since `n` is `u32` 
```suggestion
			.saturating_add(Weight::from_parts(13_002_180 / 20, 0).saturating_mul((n.into()))
```

Reply:
Done